core/src/test/java/org/opensearch/sql/analysis/NamedExpressionAnalyzerTest.java

```diff
@@ -48,4 +49,14 @@ void visit_highlight() {
     NamedExpression analyze = analyzer.analyze(alias, analysisContext);
     assertEquals("highlight(fieldA)", analyze.getNameOrAlias());
   }
+
+  @Test
+  void visit_position() {
```

Comment:
what happens when you pass it the empty string for the string pattern?  Does it return 1 in all cases or is this a failure?

Reply:
It returns 1 - the same as locate() does.